[CES (formerly Area 1)] Inform users to connect a CF account (#15525)
cloudflare/cloudflare-docs: content/email-security/account-setup/connect-cloudflare-account.md

---
title: Connect a Cloudflare account
pcx_content_type: how-to
weight: 6
---

# Connect a Cloudflare account

Existing Cloud Email Security (formerly Area 1) users must connect a Cloudflare account to access the new Zero Trust Cloud Email Security product.

> You can connect a Cloudflare account from the Cloudflare dashboard, or by connecting from the Cloud Email Security (formerly Area 1) login page.

Reviewer: ```suggestion
You can connect a Cloudflare account from the Cloudflare dashboard or from the Cloud Email Security (formerly Area 1) login page.
```

## Connect account from the dashboard

To connect a Cloudflare account from the dashboard:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/).

> 2. On the left bar, select **Area 1**.

Reviewer: ```suggestion
2. On the side bar, select **Area 1**.
```

3. Select **Launch dashboard**.

![Launch dashboard button](/images/email-security/sso/generic/launch-dashboard.png)

> 4. Select **Allow**, so that Area 1 can make changes to your Cloudflare account.

Reviewer: ```suggestion
4. Select **Allow** to enable Area 1 to make changes to your Cloudflare account.
```

![Select Allow to make changes to your Cloudflare account](/images/email-security/sso/generic/area1-allow.png)

> 5. After selecting **Allow**, log in to Area 1.

Reviewer: Step 6 is not showing below in the preview. 

<img width="623" alt="Captura de ecrã 2024-07-10, às 16 07 39" src="https://github.com/cloudflare/cloudflare-docs/assets/92738954/7862af51-70ea-4c2d-8c48-3bb857704524">

6. From here, you can either log in via [Cloudflare's Single Sign-on](https://horizon.area1security.com/), or via the **Launch dashboard** button.

## Connect account from Cloud Email Security (formerly Area 1)

1. On the [Area 1 login page](https://horizon.area1security.com/users/login), select **Sign in with Cloudflare**.

![Sign in with Cloudflare](/images/email-security/sso/generic/sign-with-cloudflare.png)
2. Log in using Cloudflare's credentials.

> 3. Select **Allow** so that Area 1 can make changes to your Cloudflare account.

Reviewer: ```suggestion
3. Select **Allow** to enable Area 1 to make changes to your Cloudflare account.
```
